.github/workflows/docker.yml

```diff
@@ -10,6 +10,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@master
+        with:
+          fetch-depth: 0
```

Comment:
Does that mean taking all history?

You could also checkout, then `git fetch --tags`, but that probably would take the same amount of time

Reply:
I think it takes the same amount of time. not worried around bandwidth right now with github actions, so think it should be fine.